doc/classes/VisualScriptSwitch.xml

```diff
@@ -1,8 +1,19 @@
 <?xml version="1.0" encoding="UTF-8" ?>
 <class name="VisualScriptSwitch" inherits="VisualScriptNode" category="Core" version="3.0.alpha.custom_build">
 	<brief_description>
+		Branches program flow based on a given input's value.
 	</brief_description>
 	<description>
+		This node branches the flow based on an input's value. Use "Case Count" to set the number of branches and each comparison's optional type. 
```

Comment:
`Branches the program flow based on an input's value. Use "Case Count" to set the number of branches and each comparison's optional type.`
What's case count?

Reply:
It's in the Inspector, I'll clarify. It's very weird there's not a member variable for this.